src/bin/elementary/meson.build

```diff
@@ -187,7 +187,11 @@ elementary_test = executable('elementary_test',
         link_args: link_args
 )
 
-library('elementary_test',
+elementary_test_lib_name = 'elementary_test'
+if sys_windows
+  elementary_test_lib_name = 'elementary_test_lib'
+endif
+library(elementary_test_lib_name,
```

Comment:
Wait...is this even used somewhere?
Shouldn't it be something like: `<lib_var> = library(...)`, so we could use it as `<lib_var>`?